tutorials/parser_low_resource/project.yml

```diff
@@ -0,0 +1,102 @@
+title: "Training a POS tagger and dependency parser for a low-resource language"
+description: |
+  This project trains a part-of-speech tagger and dependency parser for a
+  low-resource language such as Tagalog. We will be using the
+  [TRG](https://universaldependencies.org/treebanks/tl_trg/index.html) and
+  [Ugnayan](https://universaldependencies.org/treebanks/tl_ugnayan/index.html)
+  treebanks for this task. Since the number of sentences in each corpus is
+  small, we'll need to evaluate our model using [10-fold cross
+  validation](https://universaldependencies.org/release_checklist.html#data-split).
+  How to implement this split will be demonstrated in this project
+  (`scripts/kfold.py`). The cross validation results can be seen below.
+
+  |         | TOKEN_ACC | POS_ACC | MORPH_ACC | TAG_ACC | DEP_UAS | DEP_LAS |
+  |---------|-----------|---------|-----------|---------|---------|---------|
+  | TRG     | **1.000**     | **0.843**   | 0.749     | **0.833**   | *80.846**   | **0.554**   |
+  | Ugnayan | 0.998     | 0.819   | **0.995**     | 0.810   | 0.667   | 0.409   |
+
+directories: ["assets", "corpus", "training", "metrics", "configs", "models"]
+
+vars:
+  config: "default"
+  gpu_id: -1
+  n_folds: 10
+  seed: 42
+  lang: "tl"
+  trg_treebank: "UD_Tagalog-TRG"
+  trg_data: "tl_trg-ud-test"
+  ugn_treebank: "UD_Tagalog-Ugnayan"
+  ugn_data: "tl_ugnayan-ud-test"
+
+assets:
+  - dest: "assets/${vars.trg_data}.conllu"
+    description: "Treebank data for UD_Tagalog-TRG"
+    url: "https://github.com/UniversalDependencies/UD_Tagalog-TRG/blob/master/tl_trg-ud-test.conllu"
```

Comment:
Switch these to raw URLs?

Reply:
Hi, not sure if this is what we meant by raw URLs: https://github.com/explosion/projects/pull/122/commits/1d66eea9d426d2feaf1fad6745f6147a68bd6de9